packages/lab/src/Flow/stories/Usage.stories.mdx

```diff
@@ -243,7 +243,18 @@ interface HvFlowNodeSelectParam {
   options?: string[];
 }
 
-type HvFlowNodeParam = HvFlowNodeSelectParam | HvFlowNodeTextParam;
+export interface HvFlowNodeSliderParam {
+  type: "slider";
+  minPointValue: number;
+  maxPointValue: number;
+  markStep: number;
+  divisionQuantity: number;
+}
```

Comment:
We can simplify this is we point to the Slider props. Also, the `id` and `label` keys are present in the other params but not this one

Ideally, we wouldn't have to duplicate the types